src/main/java/com/ericsson/ei/erqueryservice/ERQueryService.java

```diff
@@ -131,10 +114,10 @@ public ResponseEntity getEventStreamDataById(String eventId, SearchOption search
                         .addParam("tree", Boolean.toString(tree))
                         .setBody(searchParameters.getAsJsonString(), ContentType.APPLICATION_JSON);
 
-                uri = request.getURI().toString();
+                uri = uniqRequest.getURI().toString();
                 LOGGER.debug("The URL to ER is: {}", uri);
```

Comment:
Is this logging the full request url (with parameters) ? I know we've had issues with this previously that the log only shows the base url and not the exact parameters included.

Reply:
This contains the basse url only, nothing has changed.